README.md

```diff
@@ -1,12 +1,34 @@
-# Practical case: Deployable development environment
+# Xeneta Operations Task
 
-## Premise
+The task is two-fold:
+
+* A practical case of developing a deployable development environment based on a simple application.
+
+* A theoretical case describing and evolving a data ingestion pipeline.
+
+You will be expected to present and discuss both solutions.
+
+Some general points:
+
+* **Provide the solution as a public git repository that can easily be cloned by our development team.**
+
+* Provide any instructions needed to run the automation solution in `README.md`.
+
+* The configuration file `rates/config.py` has some defaults that will most likely change depending on the solution. It would be beneficial to have a way to more dynamically pass in config values.
+
+* List and describe the tool(s) used, and why they were chosen for the task.
+
+* If you have any questions, please don't hesitate to contact us.
+
+## Practical case: Deployable development environment
+
+### Premise
 
 Provided are two simplified parts of the same application environment: A database dump and an API service. Your task is to automate setting up the development environment in a reliable and testable manner using "infrastructure as code" principles.
 
 The goal is to end up with a limited set of commands that would install the different environments and run them using containers. You can use any software that you find suitable for the task. The code should come with instructions on how to run it and deploy it to arbitrary targets; whether it is deployed locally, towards physical machines, or towards virtual nodes in the cloud.
```

Comment:
Shall we make it clear that we expect to deploy into a production environment? I mean use the words like, production/staging/QA, etc.?

Reply:
I totally missed this point on my solution

Reply:
We are asking people to make multiple development environments, not prod, staging, QA, really. It'd be natural to ask how to extend things to prod in an interview though. If we want to go another way here, we need to rework the text a bit. Should we?

Reply:
Oh, ok then my initial understanding was not incorrect then, leave it then.

Current is fine.